graph_adventure/adv.py

```diff
@@ -14,6 +14,43 @@
 
 # FILL THIS IN
 traversalPath = []
+graph = {0: {'n': '?', 's': '?', 'w': '?', 'e': '?'}}
+
+inverse_directions = {"n": "s", "s": "n", "e": "w", "w":"e"}
+
+#need to loop through this
+while True:
+    currentRoomExits = graph[player.currentRoom.id]
+
+    unexploredExits = []
```

Comment:
also have a function that can go vice versa in the directions while searching, along with an unexplored rooms functionality